packages/shorebird_cli/lib/src/commands/init_command.dart

```diff
@@ -315,7 +315,7 @@ app_id:
     return ShorebirdYaml(appId: appId);
   }
 
-  void _addShorebirdYamlToPubspecAssets(File pubspecFile) {
+  static void addShorebirdYamlToPubspecAssets(File pubspecFile) {
```

Comment:
If this is being used outside of this command, it should probably live somewhere else (possibly in ShorebirdEnv), and be referenced by this command and the new validator.